plutus-core/untyped-plutus-core/test/Evaluation/FreeVars.hs

```diff
@@ -1,6 +1,7 @@
 -- editorconfig-checker-disable-file
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies     #-}
+{-# LANGUAGE TypeOperators    #-}
```

Comment:
You know, maybe we should just give in and use some `default-extensions` already.